urdf/scripts/mesh_distortion/README.md

```diff
@@ -33,22 +33,30 @@ There are two Coke can models as examples:
 `models/dave_object_models/models/Coke_Can/meshes/coke_can.obj`.
 Choose Transform while importing: Z Up, X Forward.
 
+Tip: To view the textured object in the viewport, click on the globe icon in the
+upper right of the viewport, which is for Viewport Shading: Material Preview.
+
 Look for the mesh name in the model.
```

Comment:
Minor suggestion,
I was trying to use custom 'object_prefix' other than 'LPCoke_Cube'. Failed, of course. The naming 'prefix' gives me a feeling that its customizable (got lazy to type 'LPCoke_Cube' and typed 'test' instead). Wouldn't it be better to change `object_prefix='LPCode_Cube'` to `object_name='LPCode_Cube'`?

Reply:
There is a specific reason for it to be called `prefix`. If you run the script multiple times, the mesh objects will be imported with the same name, but with incremented suffix `.001`, `.002`, `.003`, etc. (I don't know why Blender does that. It does that if you do it manually too, even if you delete all the meshes in the scene.)

So we cannot just look for an exact match in the object name. We have to look for the prefix. The Python line that looks for the object uses `startswith()`.

Reply:
Understood. I should learn `startswith()`